CONTRIBUTING.md

```diff
@@ -174,7 +174,6 @@ Here are the used scopes under 3 different markdown flavors. If you are not sure
 
 ### ScopeName: text.html.markdown.multimarkdown
 
-* <All scopes under text.html.markdown.gfm>
```

Comment:
It should be `* < All scopes under text.html.markdown.gfm >` to make it visible.